en/config_heli/README.md

```diff
@@ -1,31 +1,37 @@
 # Helicopter Configuration
 
-This section contains topics related to helicopter configuration and tuning.
-
-:::note
-Helicopters are less well-supported on than other types of vehicles.
-For example, PX4 does not support helicopters with coaxial or dual rotor types, and features such as RPM governor and autorotation.
-We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
-:::
+This section contains topics related to [helicopter](../frames_helicopter/README.md) configuration and tuning.
 
 ## Supported Configurations
 
 Supported helicopter configurations:
 
 - Single main rotor with swash-plate controlled by up to 4 swash-plate servos and a mechanically uncoupled tail rotor driven by an ESC.
+- Single main rotor with swash-plate controlled by up to 4 swash-plate servos and a mechanically coupled tail controlled by a servo.
+  (This can be flown by mapping the tail servo to the tail motor output and setting an adequate range and disarmed value for the servo.
+  The system currently assumes it's a separate motor.)
 
-:::note
-Mechanically coupled tails controlled by a servo can be flown by mapping the tail servo to the tail motor output and setting an adequate range and disarmed value for the servo. But the system currently assumes it's a separate motor.
-:::
+Supported flight operations/features:
 
-Supported flight opertaion:
-- Same as a multicopter. At the time of writing no autonomous/guided 3D flying with negative thrust is possible.
+- Same as a multicopter.
+- At the time of writing no autonomous/guided 3D flying with negative thrust is possible.
 
 ## Setup
 
 To setup and configure a helicopter:
 
-1. Select the helicopter [Airframe](../config/airframe.md) in QGroundControl
+1. Select a helicopter [Airframe](../config/airframe.md) in QGroundControl.
+   At time of writing there is only _Generic Helicopter (Tail ESC)_ in the Helicopter group.
+   This will configure the helicopter frame with a mechanically uncoupled tail (tail ESC).
+
+   ![QGC - helicopter airframe](../../assets/config/airframe/airframe_heli_generic.png)
+
+   :::note
+   There is no separate airframe for the helicopter with tail servo.
+   To select this configuration, set the parameter [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME) to _Helicopter (tail Servo)_.
+   The actuator configuration screen will then change to support this frame type.
+   :::
```

Comment:
@bkueng I updated this to refer to changing CA_AIRFRAME for the tail servo case - can't check it - having trouble getting my firmware to update to main. 

Can you read this section lines 10 to 333, and in particular the paragraph on line 11. I am not sure if that is correct, or has been superseded.

I have made an assumption here that most of the config using helicopter with a tail servo is the very similar to existing setup.

I'm going to merge. I hope this is OK and can easily be fixed if not.